v1.1.1/index.html

```diff
@@ -38,7 +38,7 @@
 
 	<!-- Loopy & Ink -->
 	<div id="canvasses"></div>
-	<div id="graph_div"></div>
+	<div id="graph_div" style="top: 10px; left: 100px;"></div>
```

Comment:
Changing this line to read `<div id="graph_div" style="top: 10px; left: 100px;" onclick="subscribe('mouseclick');"></div>` like I discussed here https://github.com/nlehnert1/loopy/pull/63/files#r811265754 does indeed work, but it makes it so that the graph sidebar gets displayed on click and drag as well as just click. I'm not sure that's the behavior I want honestly, what are your thoughts?